### PR TITLE
RM-7878 The border of BocTextValues & BocMultilineTextValues is cut off in Firefox 89

### PR DESCRIPTION
**Pull request by raffaelfoidl**
https://re-motion.atlassian.net/browse/RM-7878

### CODE REVIEW

**Comment by raffaelfoidl** [comment]
The modified Screenshots were simply adjusted to the slightly changed layout:
image
image

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css [hunk 1]
```diff
@@ -12,6 +12,10 @@ span.bocDateValue
   white-space: nowrap;
 }
 
+.mozilla span.bocDateInputWrapper {
+    margin-right: 2px;
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css:16]:
```suggestion
.mozilla span.bocDateInputWrapper
{
    margin-right: 2px;
```

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css [hunk 2]
```diff
@@ -10,6 +10,11 @@ span.bocAutoCompleteReferenceValue
   min-height: 1.7em;
 }
 
+.mozilla span.bocAutoCompleteReferenceValueInput
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css:13]: please move down to the other `...Input` entries.

I'll also need to understand the use of margin vs. width vs. padding here. When dealing with inline, margin has very specific semantics and we haven't used it with input elements before so I need to understand why this works compared to other approaches.

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css [hunk 3]
```diff
@@ -12,6 +12,10 @@ span.bocDateValue
   white-space: nowrap;
 }
 
+.mozilla span.bocDateInputWrapper {
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css:15]: Please move down to the other `bocDateInputWrapper`.

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

**Comment by raffaelfoidl** [comment]
Text:
image

MultiLineText:
image

DateTime:
image

AutoComplete:
image

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css [hunk 4]
```diff
@@ -35,6 +35,11 @@ span.bocDateValue.disabled
 {
 }
 
+.mozilla span.bocDateInputWrapper
+{
+    margin-right: 2px;
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css:40]: There's a margin-right here.

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**